common/src/sports-info.ts

```diff
@@ -0,0 +1,10 @@
+export interface Fixture {
+  idEvent: string
+  strLeague: string
+  strEvent: string
+  strHomeTeam: string
```

Comment:
don't prefix with str, that's redundant. Just call each by their name, e.g. `league`